backend-js/src/routes/items.js

```diff
@@ -1,11 +1,57 @@
+import S from "fluent-json-schema";
+
 export default async (server, _, done) => {
-  server.get("/items/:id", async (request) => {
-    const id = request.params.id;
-    const client = await server.pg.connect();
-    const { rows } = await client.query(`SELECT * FROM "Item" WHERE id=${id}`);
-    client.release();
-    return rows;
-  });
+  server.get(
+    "/items/:id",
+    {
+      schema: {
+        tags: ["items"],
+        params: {
+          type: "object",
+          properties: {
+            id: {
+              type: "number",
+            },
+          },
+        },
+        response: {
+          200: S.object().prop("id", S.number()).prop("name", S.string()),
+        },
+      },
+    },
+    async (req, reply) => {
+      const { id } = req.params;
+      const client = await server.pg.connect();
+      const { rows } = await client.query(
+        `SELECT * FROM "Item" WHERE id=${id}`
+      );
+      client.release();
+      if (!!!rows.length) reply.send(server.httpErrors.badRequest());
+      return rows[0];
+    }
+  );
+
+  server.post(
+    "/items",
+    {
+      schema: {
+        body: S.object().prop("data", S.object().prop("name", S.string())),
+        response: {
+          201: S.object().prop("created", S.boolean()),
+        },
+      },
+    },
+    async (req, reply) => {
+      const { name } = req.body;
+      const client = await server.pg.connect();
+      const id = await client.query(
+        'INSERT INTO "Item"(name) VALUES($1) RETURNING id',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexanderdavide/fastify-react-example/pullRequest?prid=7390353)
```suggestion
        "INSERT INTO \"Item\"(name) VALUES($1) RETURNING id",
```